Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Copy content using te role of Editor
+    In order to validate the copy action
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        When I copy the "News Flash" into "Destiny" folder
+        Then the "News Flash" is copied with message "'News Flash' has been successfully copied under 'Destiny'"
+        And I see "News Flash" as a child of "Destiny" folder
+
+    @javascript
+    Scenario: Copy one object that has children objects
+        Given an "Destiny" folder exists
+        And an "Origin" folder exists
+        And a "News Flash" article exists as a child of "Origin" folder
+        When I copy the "News Flash" into "Destiny" folder
+        Then the "News Flash" is copied
+        And I see "News Flash" as a child of "Origin" folder
+        And I see "News Flash" as a child of "Destiny" folder
+
+    @javascript
+    Scenario: Content tree is updated after the copy of an object
+        Given an "Destiny" folder exists
```

Comment:
I'm thinking that this is gonna happen a lot (folder/folder/article). I'd almost suggest you add a shortcut sentence:

```
Given that an article "Origin/News Flash" exists
```

The leaf in the sentence would be of the given content-type (article), and all the parents would just be folders. It's kind of how we refer to content when talking about issues, if you think about it.